parallel-install/pkg/preinstaller/output.go

```diff
@@ -0,0 +1,14 @@
+package preinstaller
```

Comment:
These types are used only by things in "preinstaller.go". I think it makes sense to move them there.